ocrd_utils/ocrd_utils/__init__.py

```diff
@@ -5,29 +5,48 @@
 
```

Comment:
Please add more description items __above__ (being more abstract than the converter functions) the first item on line 4.
```suggestion
* coordinates_of_segment, coordinates_for_segment, rotate_coordinates

These functions convert polygon outlines for PAGE elements on all hierarchy levels (page, region, line, word, glyph) between relative coordinates w.r.t. parent segment and absolute coordinates w.r.t. the top-level (source) image. This includes rotation and offset correction.

* polygon_mask, image_from_polygon, crop_image

These functions combine PIL.Image with polygons or bboxes.
```

Reply:
GH does not allow me to make suggestions outside the 3 lines of context within existing hunks. Please read my original comment: this docstring is meant as the _first item_ on __line 4__. 

Also, GH does not offer a proper interface to calculate suggestions from the result of proper editing, so my suggestions were necessarily incomplete. Please have a look at the final result – I see missing indentation and quotation, suboptimal newlines etc.

Reply:
@kba What of it now – you already merged, despite these unresolved – should I make a new PR with the docstrings fixed?

Reply:
> @kba What of it now – you already merged, despite these unresolved – should I make a new PR with the docstrings fixed?

I thought I had resolved the issues in 677890b and ac95d38. Can you make a PR against master with the fixed docstrings or at least a WIP PR which shows what you want changed? Thanks!

Reply:
Yes, I will do that.